Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/Indexer.java

```diff
@@ -60,23 +60,20 @@ public Indexer(String type) {
      *            bean object which will be added or deleted from index
      * @param baseType
      *            type on which will be called method createDocument()
-     * @return response from the server
      */
     @SuppressWarnings("unchecked")
-    public String performSingleRequest(T baseIndexedBean, S baseType) throws IOException, CustomResponseException {
+    public void performSingleRequest(T baseIndexedBean, S baseType) throws IOException, CustomResponseException {
         IndexRestClient restClient = initiateRestClient();
-        String response;
 
         if (method == HTTPMethods.PUT) {
             HttpEntity document = baseType.createDocument(baseIndexedBean);
-            response = String.valueOf(restClient.addDocument(document, baseIndexedBean.getId()));
+            restClient.addDocument(document, baseIndexedBean.getId());
```

Comment:
Why are you ignoring return value of `addDocument` method? Previous behaviour was even wrong as this method is returning a boolean value, but this is an other story.

Reply:
Boolean value is only useful for testing purpose. This whole string return is useless here. We should be notified if something goes wrong by exceptions, not return values.

Reply:
Any static code analyse tool will report this ignored return value from a method as bad programming.

Reply:
Boolean is not returned anymore.